packages/react/src/components/DetailsList/DetailsRow.base.tsx

```diff
@@ -214,6 +215,8 @@ export class DetailsRowBase extends React.Component<IDetailsRowBaseProps, IDetai
     const isContentUnselectable = selectionMode === SelectionMode.multiple;
     const showCheckbox = selectionMode !== SelectionMode.none && checkboxVisibility !== CheckboxVisibility.hidden;
     const ariaSelected = selectionMode === SelectionMode.none ? undefined : isSelected;
+    const group = groups && getItemGroup(groups, itemIndex);
+    const ariaPositionInSet = groups && group ? ((itemIndex - group.startIndex) % groups.length) + 1 : undefined;
```

Comment:
This only happens to work in our example page b/c of the way we generate groups and rows, where each group always has the same number of rows, which is also the same as the number of groups in each level. If you change GroupedList.Basic.Example.ts to have 4 groups of 7 rows each, you can see it fall down.

It looks like the issue is that you're getting the top-level group for the item, rather than the immediate group. If you use `groupNestingDepth` to find the immediate parent group you should be able to do `itemIndex - group.startIndex + 1`.

I know it's more group iteration and not particularly pretty, unless there's some easier helper to get the immediate group parent that I'm not seeing immediately.

Reply:
I'll modify the `getItemGroup` helper function to dig deeper into the nested groups to get the immediate parent. One other option I could have done was to pass down the exact row `group` variable from `GroupedListSection` but that would have required at least 3-4 changes in various onRenderCell functions (in different components) just to pass that parameter which seemed too intrusive in comparison to just passing `groups` as a prop to `DetailsRow` and letting that component figure out the rest 😅